src/Microsoft.AspNetCore.ResponseCaching/ResponseCacheOptions.cs

```diff
@@ -14,14 +14,15 @@ public class ResponseCacheOptions
         public long MaximumCachedBodySize { get; set; } = 1024 * 1024;
 
         /// <summary>
-        /// <c>true</c> if request paths are case-sensitive; otherwise <c>false</c>. The default is to treat paths as case-insensitive.
+        /// The shard size for the response body in bytes. The default is set to 1 KB.
         /// </summary>
-        public bool UseCaseSensitivePaths { get; set; } = false;
+        // TODO: Setting to 1KB for testing, 75 KB is more reasonable.
+        public int CachedBodyShardSize { get; set; } = 10;
```

Comment:
Was 10k until I wanted to force sharding in testing. I often use GUIDs for response bodies so I needed a smaller shard size. That said, I only opened this PR to show what sharding with streams look like, there are tons of minor details that still need to be polished and it's not really ready for final reviewing just yet.